engine/dlib/src/dlib/buffer.cpp

```diff
@@ -397,6 +403,11 @@ namespace dmBuffer
         buffer->m_Data = (void*)((uintptr_t)data_block + header_size);
         buffer->m_Stride = struct_size;
         buffer->m_ContentVersion = 0;
+        // meta data
+        buffer->boundsMin = dmVMath::Point3(1.0f, 1.0f, 1.0f);  // initialize to an "invalid" bounding box
+        buffer->boundsMax = dmVMath::Point3(-1.0f, -1.0f, -1.0f);
+        buffer->m_BoundingVersion = 999; // any value different than m_ContentVersion will allow triggering first bounds calculation
```

Comment:
Perhaps set it to `contentversion + 1` to make it clear.

Reply:
👍

Reply:
Yeah, it was my first thought too. 

But then i thought that when the buffer is updated and m_ContentVersion gets incremented (becomes 1) we'll have a case where `m_BoundingVersion  == m_ContentVersion` which is semantically wrong. Also, '-1' won't work since it's unisgned. 

I ended up using 999.

Reply:
Another idea was to initialize set m_ContentVersion to 1 and m_BoundingVersion to 0. Semantically it seems the best. But i worried about breaking something.